templates/docs/patterns/strip/index.md

```diff
@@ -84,43 +60,26 @@ This state gives the strip smaller vertical padding.
 View example of the pattern strip is-shallow
 </a></div>
 
-## Suru strip
+## Deprecated
 
-<span class="p-status-label--negative">Deprecated</span>
-
-<div class="p-notification--negative">
+<div class="p-notification--caution">
   <div class="p-notification__content">
-    <h5 class="p-notification__title">Deprecated</h5>
-    <p class="p-notification__message">Strips with old style of the Suru are now deprecated and should not be used on any new pages. Use a blank strip or <a href="/docs/patterns/suru">new Suru component</a> instead.</p>
+    <h3 class="p-notification__title">Deprecated</h3>
+    <p class="p-notification__message">In Vanilla 4.8.0 with the introduction of new theming system and updated Suru component various legacy strip variants have been deprecated.</p>
   </div>
 </div>
 
-This is a patterned strip that is ideal for overview or main pages, and can be used with images.
+The following strip variants are now deprecated and should not be used on any new pages:
 
-The colours of the solid gradient are based on `$color-brand` by default. The gradient colours can be customised by overriding the `$color-suru-start`, `$color-suru-middle` and `$color-suru-end` variables. A dark colour scheme is recommended, as the text colour is light by default.
+Instead of deprecated [Suru strip](/docs/examples/patterns/strips/suru/) (`.p-strip--suru`) and [topped Suru strip](/docs/examples/patterns/strips/suru-topped/) (`.p-strip--suru-topped`) use the new [Suru component](/docs/patterns/suru) instead.
 
-<div class="embedded-example"><a href="/docs/examples/patterns/strips/suru/" class="js-example">
-View example of the Suru strip pattern
-</a></div>
+[Strips with arbitrary image backrounds](/docs/examples/patterns/strips/image/) (`.p-strip--image`) are now also deprecated. For a hero section with a background use the new Suru component instead.
 
-## Topped Suru strip
+[Bordered strip](/docs/examples/patterns/strips/is-bordered/) (`.p-strip--bordered`) is now deprecated. If a horizontal line is needed to separate parts of content use standard [sections](/docs/patterns/section) and [the rule component](/docs/patterns/rule) instead.
 
-<span class="p-status-label--negative">Deprecated</span>
+[Light strips](/docs/examples/patterns/strips/strips-light) (`.p-strip--light`), [white strips](/docs/examples/patterns/strips/white) (`.p-strip--white`) and [accent strips](/docs/examples/patterns/strips/accent) (`.p-strip--accent`) are now deprecated. Use the new [highlighted strip](#highligted-strip) instead.
```

Comment:
There's a minor typo for #highligted-strip, causing the jump to header to not work

Reply:
You are so good at this!